### ISSUE TITLE
Why is aria-placeholder not supported by the ARIA combobox role?

### ISSUE THREAD

**Issue by kloots** [jsx-eslint/eslint-plugin-jsx-a11y#783]
The linter indicates the `aria-placeholder` attribute is not supported by the ARIA `combobox` role. However:

- [The ARIA spec for `aria-placeholder`](https://www.w3.org/TR/wai-aria-1.1/#aria-placeholder) doesn't indicate it cannot be used with the `combobox` role.
- [The ARIA spec for the `combobox` role](https://www.w3.org/TR/wai-aria-1.1/#combobox) doesn't indicate it cannot be used with `aria-placeholder`
- Testing across browsers + screen readers reveals `aria-placeholder` does appear to be supported.

Wondering if this is a false negative. Or if it isn't: why `aria-placeholder` is considered incompatible with the `combobox` role?
